addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/handler/HomematicBridgeHandler.java

```diff
@@ -238,7 +238,7 @@ public void onStateUpdated(HmDatapoint dp) {
         Thing hmThing = getThingByUID(UidUtils.generateThingUID(dp.getChannel().getDevice(), getThing()));
         if (hmThing != null && hmThing.getHandler() != null) {
             final ThingStatus status = hmThing.getStatus();
-            if (status == ThingStatus.ONLINE || status == ThingStatus.OFFLINE) {
+            if (status != null && (status == ThingStatus.ONLINE || status == ThingStatus.OFFLINE)) {
```

Comment:
my fault, i saw a NPE but i had the wrong source code. The NPE has already been fixed in a previous PR